src/MSBuild.Bootstrap/MSBuild.Bootstrap.csproj

```diff
@@ -48,6 +48,12 @@
     <Copy SourceFiles="$(PublishDir)$(AssemblyName).deps.json" DestinationFiles="$(PublishDir)MSBuild.deps.json" />
   </Target>
 
+  <PropertyGroup>
+    <IgnoreMSBuildTaskHost>true</IgnoreMSBuildTaskHost>
```

Comment:
Why though?

Reply:
It failed to build due to some obscure nuget tfm compat reason, and since MSBuildTaskHost wasn't required for the cloudbuild scenario, I decided to just disable it, with the reasoning that whomever wants 64bit task host can debug that and enable it :)

Reply:
Looks good. Simpler fix than what the daunting error message suggested :)